### PR TITLE
Split CSS into multiple files

### LINKED ISSUE #212: Split CSS into multiple files
Our CSS would be better organized in multiple files based on the current comments indicating what function each part of it serves.

### PR DESCRIPTION
**Pull request by diogoscf**
Fix #212 by splitting css into multiple files and changing other files accordingly

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -179,6 +179,8 @@ $(THEMES): % : all build/css/bootstrap.%.css build/css/.d
 
 theme = $(filter $(THEMES), $(MAKECMDGOALS))
 
+CSSFILES := assets/css/analysis.css assets/css/footer.css assets/css/general.css assets/css/navbar.css assets/css/nojs.css assets/css/rtl.css assets/css/translation.css
```
Comment (sushain97) [inline Makefile]: Same formatting as `JSFILES` and right underneath the section comment, please.

File: assets/css/general.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    position: absolute;
+    top: 45%;
+    z-index: 10000;
+}
+
+.chaining {
+    display: none;
+}
+
+.localeGlobe {
```
Comment (sushain97) [inline assets/css/general.css]: Hm, do you think this goes in `navbar` now?

File: assets/css/general.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+#loadingIndicator {
+    display: none;
+    left: 48%;
+    position: absolute;
+    top: 45%;
+    z-index: 10000;
+}
+
+.chaining {
```
Comment (sushain97) [inline assets/css/general.css]: I think this goes in translation?

File: assets/css/nojs.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    z-index: 10000000000000;
+}
+
+.navbar {
+    margin-top: 4em;
+}
+
+body > .container {
+    margin-top: 4em;
+}
```
Comment (sushain97) [inline assets/css/nojs.css]: I think these can all just go in `general` :)

File: assets/css/rtl.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+html[dir='rtl'] .pull-right:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
```
Comment (sushain97) [inline assets/css/rtl.css]: These can go in `general` as well.

File: assets/css/nojs.css [hunk 6]
```diff
@@ -0,0 +1,16 @@
+#noscript {
+    position: fixed;
+    right: 0;
+    text-align: center;
+    top: 5px;
+    width: 100%;
+    z-index: 10000000000000;
+}
+
+.navbar {
```
Comment (sushain97) [inline assets/css/nojs.css]: This one specifically might go in navbar?

File: debug-head.html [hunk 7]
```diff
@@ -1,5 +1,11 @@
         <link type="text/css" rel="stylesheet" href="./css/bootstrap.css">
-        <link type="text/css" rel="stylesheet" href="./css/style.css">
+        <link type="text/css" rel="stylesheet" href="./css/analysis.css">
+        <link type="text/css" rel="stylesheet" href="./css/footer.css">
+        <link type="text/css" rel="stylesheet" href="./css/general.css">
+        <link type="text/css" rel="stylesheet" href="./css/navbar.css">
+        <link type="text/css" rel="stylesheet" href="./css/nojs.css">
+        <link type="text/css" rel="stylesheet" href="./css/rtl.css">
+        <link type="text/css" rel="stylesheet" href="./css/translation.css">
```
Comment (sushain97) [inline debug-head.html]: Could you order these a bit more? general, navbar, footer, rtl, translation, analysis makes sense to me.

**Comment by diogoscf** [comment]
I’ll fix all of that later today. I had just used the comments to sort out the css files and alphabetical order in debug-head

File: assets/css/nojs.css (hunk 4 above)
Comment (diogoscf) [reply assets/css/nojs.css]: So all of `nojs` into `general` except the rule-set for `.navbar`?

File: assets/css/general.css (hunk 2 above)
Comment (diogoscf) [reply assets/css/general.css]: Yeah probably

Comment (diogoscf) [reply assets/css/general.css]: It's also used in the main part of the page, but I think it's hidden, so probably yes

Comment (diogoscf) [reply assets/css/general.css]: And same for localeSelect below

**Pull request closed without merging**